sympy/stats/crv_types.py

```diff
@@ -235,6 +235,8 @@ def pdf(self, x):
         return (exp(-alpha*log(x/sigma) - beta*log(x/sigma)**2)
                *(alpha/x + 2*beta*log(x/sigma)/x))
 
+    def _moment_generating_function(self, t):
+        raise NotImplementedError('The moment generating function of the Benini distribution does not exist.')
```

Comment:
This line is too long. Please split.
"long string" can be written 
("long "
  "string")